### PR TITLE
Update mruby to 78add728126cbe03b9048e57f897a7134508af5d

### PR DESCRIPTION
**Pull request by craiglrock**

### CODE REVIEW

File: spec/mruby_engine_spec.rb [hunk 1]
```diff
@@ -147,9 +147,7 @@ def bar
       SOURCE
     }.to raise_error(MRubyEngine::EngineRuntimeError) do |e|
       expect(e.guest_backtrace).to eq([
-        "backtrace.rb:2:in Object.foo",
-        "backtrace.rb:6:in Object.bar",
-        "backtrace.rb:9",
+        "backtrace.rb:2:in Object.foo"
```
Comment (alexsnaps) [inline spec/mruby_engine_spec.rb]: Are we missing stack frames now?!

Comment (craiglrock) [inline spec/mruby_engine_spec.rb]: yeah that is a bit odd, I'll see if I can nail down what's causing it.

Comment (alexsnaps) [inline spec/mruby_engine_spec.rb]: Not that it matters much _here_ though...

**Comment by craiglrock** [comment]
Proposed fix for the backtrace issue, https://github.com/mruby/mruby/pull/3659

**Comment by alexsnaps** [comment]
👍

**Pull request merged**